### PR TITLE
keep appliedmanifestwork & clusterclaim CRDs when unstalling klusterlet

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go [hunk 1]
```diff
@@ -712,20 +712,6 @@ func (n *klusterletController) cleanUp(
 		}
 	}
 
-	// remove the CRDs
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:715]: do we need to remove the CR?

Comment (elgnay) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:715]: As we discussed in slack, it's fine to leave the CRs for now.

Comment (qiujian16) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:715]: please add a comment here indicating we do not delete crd and CR and the reason.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go (hunk 1 above)
Comment (elgnay) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:715]: Comment added.

**Pull request merged**